src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageViewModel.cs

```diff
@@ -296,6 +296,38 @@ public bool HasLaunchOption
             }
         }
 
+        public bool NativeCodeDebugging
+        {
+            get
+            {
+                if (!IsProfileSelected)
+                {
+                    return false;
+                }
+
+                if (SelectedDebugProfile.OtherSettings.Keys.Contains("nativeDebugging"))
+                {
+                    return (bool) SelectedDebugProfile.OtherSettings["nativeDebugging"];
```

Comment:
Nit: I don't think we have spaces after casts.. could be missing from .editorconfig?